modules/serverless-logic-building-deploying-workflow-preview-mode.adoc

```diff
@@ -112,5 +69,13 @@ Ensure that the pod corresponding to your workflow is running.
 +
 [source,terminal]
 ----
-$ oc logs pod/<pod-name> -n workflows
+$ oc logs pod/<pod-name> -n <namespace>
+----
+
+.Cleanup
```

Comment:
Technically `.Cleanup` is a non-standard heading in a procedure module (per [modular docs guide](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs)). FWIW, I see only 1 other instance in the repo (that one has it as `.Clean up` two-words, tho the single-word here is correctly what IBM SG suggests).

Could this instead just be the last step in the normal `.Procedure`? Or maybe the last step in `.Verification`? Alternatively, could be its own procedure "Cleaning up a workflow deployment" but that seems like overkill.